src/agat/agat_sp_merge_annotations/test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+test_dir="${meta_resources_dir}/test_data"
+out_dir="${meta_resources_dir}/out_data"
```

Comment:
Could you use a temporary directory for the test data output?
Something similar to this: 
```
# create temporary directory and clean up on exit
TMPDIR=$(mktemp -d --tmpdir "$meta_temp_dir")
function clean_up {
    rm -rf "$TMP_DIR"
}
trap clean_up EXIT
```